pydm/utilities/iconfont.py

```diff
@@ -21,7 +21,7 @@ class IconFont(object):
     def __init__(self):
         if self.__initialized:
             return
-        self.font_file = "fontawesome.ttf"  # specify these relative to this file.
+        self.font_file = "fontawesome.ttf" # specify these relative to this file.
```

Comment:
Think this was intended to have two spaces per [PEP8](https://lintlyci.github.io/Flake8Rules/rules/E261.html)